Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -164,7 +169,11 @@ fi
 
 # Check for graphical package for MAVProxy
 if [[ $SKIP_AP_GRAPHIC_ENV -ne 1 ]]; then
-  if [ ${RELEASE_CODENAME} == 'focal' ]; then
+  if [ ${RELEASE_CODENAME} == 'groovy' ]; then
+    SITL_PKGS+=" python3-wxgtk4.0"
+    SITL_PKGS+=" fonts-freefont-ttf libfreetype6-dev libjpeg8-dev libpng16-16 libportmidi-dev libsdl-image1.2-dev libsdl-mixer1.2-dev libsdl-ttf2.0-dev libsdl1.2-dev"  # for pygame
+    SITL_PKGS+=" python-is-python3"
```

Comment:
this need to be on the if line 163, otherwise docker images failed at using waf

Reply:
OK, moving that.